docker/windows/Scripts/configure-tentacle.ps1

```diff
@@ -112,7 +113,11 @@ function Validate-Variables() {
   Write-Log " - server endpoint '$ServerUrl'"
   Write-Log " - api key '##########'"
   if (($null -ne $ServerCommsAddress) -or ($null -ne $ServerPort)) {
-    Write-Log " - communication mode 'Polling' (Active)"
+    if ($null -ne $AsKubernetesTentacle ) {
+      Write-Log " - communication mode 'Kubernetes' (Polling)"
```

Comment:
Why are we adding this?

Reply:
I didn't... I suspect this is a rebase/merging issue.

I'll sort this out when the branch it's going into gets merged into master.